templates/typescript-library/tsup.dev.ts

```diff
@@ -0,0 +1,5 @@
+import { defineDevConfig } from "@workleap/tsup-configs";
+
+export default defineDevConfig({
+    clean: true
```

Comment:
For the dev config, i'm not sure it's best to clean. People usually use it to "watch" the files, and you don't want to always clean on every changes